### PR TITLE
Documentation Guide and Structure

### PR DESCRIPTION
**Pull request by tyg**
Description of our documentation structure and a suggested reading order.

### CODE REVIEW

**Comment by katelynsills** [comment]
This looks great to me! What is the next step? Seems like if we added links to this document (some links might still be TODOs), we can turn this into a real PR. 

We have two options for an unfinished doc where the links might be TODOs: 1) we can merge it into master and publish, but just not add it to the menus, so it will not be noticeable, or 2) we can publish it and add it to the menus even with the TODOs, since it will probably be an improvement.

File: main/getting-started/Documentation-Map.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
+  software.
+3. **Introduction to Zoe**: A Getting Started document that
+  briefly explains the concepts and components of Zoe, our platform
+  for writing smart contracts in JavaScript. Zoe implements our smart
+  contract guarantee that users either get what they wanted from the
+  contract or a full refund of what they put up for it. Regardless of
+  whether the contract is buggy or malicious.
```
Comment (Chris-Hibbert) [inline main/getting-started/Documentation-Map.md]: ==> "or a full refund of what they put up for it, regardless of whether the contract is honest, buggy or malicious."

File: main/getting-started/Documentation-Map.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  even copy code from for use in your projects. This document shows
+  you how to install and modify the AutoSwap demo contract.
+5. **ERTP Guide**: A detailed description of ERTP concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+6. **Zoe Guide**: A detailed description of Zoe concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+7. **Creating New Smart Contracts and Using Existing Ones**: A
+  procedure checklist table Reference document that walks you through
```
Comment (Chris-Hibbert) [inline main/getting-started/Documentation-Map.md]: I like the form of your "procedure checklist table Reference document", but I sure don't like having to refer to it that way. I think "procedure checklist" is plenty to convey the idea.

File: main/getting-started/Documentation-Map.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- **Reference**: These are the specs for our APIs and commands,
+  as well as procedure checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
```
Comment (Chris-Hibbert) [inline main/getting-started/Documentation-Map.md]: Presumably these will all be links before being publishedd.

**Comment by DavidBruant** [comment]
Sharing an info in case it's useful: 
Markdown allows HTML comments, so it's possible to do something like

```md
**some markdown content**

<!--
TODO add the third item
-->

- first item
- second item
```

Same content, but rendered

---
**some markdown content**



- first item
- second item
---

The comment is present in the markdown file, but not in the rendered HTML

File: main/getting-started/Documentation-Map.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+read. They cover what to do to install, set up, and start
+using Agoric's smart contract tools. They also introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: Should we imply that installing Agoric software is an extended procedure? Maybe: "and explain the procedure for tasks such as installing and using the Agoric software."

File: main/getting-started/Documentation-Map.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: Do we need to explain this part? It seems a little nitpicky

File: main/getting-started/Documentation-Map.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
+   Agoric software and installing the Agoric SDK.
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: Maybe instead: "at the heart of Agoric software"? At the base implies the tech stack to me, but ERTP is near the top of the stock.

File: main/getting-started/Documentation-Map.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  software.
+3. **Introduction to Zoe**: A Getting Started document that
+  briefly explains the concepts and components of Zoe, our platform
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: This is beyond this text, but I think when we say "platform" people think it is another blockchain. I wonder if we can come up with a term to explain that it is a reusable smart contract that runs other smart contracts.

File: main/getting-started/Documentation-Map.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+  for writing smart contracts in JavaScript. Zoe implements our smart
+  contract guarantee that users either get what they wanted from the
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: "smart contract guarantee" sounds like it isn't saying much. Should we name offer safety here? Or just use the word "guarantee" alone?

File: main/getting-started/Documentation-Map.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+  contract or a full refund of what they put up for it. Regardless of
+  whether the contract is honest, buggy, or malicious.
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: Not a full sentence, but maybe that is ok for emphasis?

File: main/getting-started/Documentation-Map.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+4. **Using An Agoric Demo Contract**: Agoric supplies several
+  demo smart contracts for you to experiment with, learn from, and
+  even copy code from for use in your projects. This document shows
+  you how to install and modify the Autoswap demo contract.
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: I think this should be Simple Exchange, not Autoswap.

File: main/getting-started/Documentation-Map.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+5. **ERTP Guide**: A detailed description of ERTP concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+6. **Zoe Guide**: A detailed description of Zoe concepts,
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: I think if we can describe both guides using the same words, it is too vague. Maybe we can include more ERTP or Zoe specific concepts?

File: main/getting-started/Documentation-Map.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+  design, components, and commands. Includes examples of command 
+  usage.
+7. **Creating New Smart Contracts and Using Existing Ones**: A
+  procedure checklist that walks you through
+  creating a new smart contract from
+  scratch. Another procedure checklist walks you through editing
+  and using an existing contract.
+
+When needed, refer to the **ERTP API Reference**, **Zoe API
+Reference**, and **Agoric CLI Reference** for command specifications.
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: "command specifications" is a new phrase for me.

File: main/getting-started/Documentation-Map.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+We also have an **Agoric Glossary** for terms we've given Agoric-context
+definitions to (i.e. what does *mint* mean in an Agoric context?).
+
+For more about Agoric's ideas, plans, and goals, see our **Learn More**
+page. It has links to Agoric's company homepage and various documents and 
+videos you may find useful, such as conference talks, white papers, 
+conference papers, etc.
+
+Finally, we have links to the ultimate documentation; the GitHub
+repositories for the code that defines the Agoric SDK and our Zoe
```
Comment (katelynsills) [inline main/getting-started/Documentation-Map.md]: This division doesn't necessarily make sense to me. ERTP and Zoe are equally part of the Agoric SDK repo. Are we highlighting Zoe for emphasis?

**Pull request merged**